articles/openstack/ostack-how-basic-heat-template.md

```diff
@@ -0,0 +1,164 @@
+---
+title: How to create and deploy a basic heat template  | UKCloud Ltd
```

Comment:
Quick google seems to suggest that Heat should be init capped (we should use whatever capitalisation OpenStack uses). So change throughout.